libs/feature/editor/src/lib/+state/editor.selectors.spec.ts

```diff
@@ -36,5 +38,32 @@ describe('Editor Selectors', () => {
       const result = EditorSelectors.selectRecordChangedSinceSave(state)
       expect(result).toBe(true)
     })
+
+    it('selectRecordFieldsConfig() should return the current "fieldsConfig" state', () => {
+      const result = EditorSelectors.selectRecordFieldsConfig(state)
+      expect(result).toEqual(DEFAULT_FIELDS)
+    })
+
+    it('selectRecordFields() should return the current "fieldsConfig" state', () => {
```

Comment:
Same "should" as above?

Reply:
I used copilot a bit too much :grimacing: